src/components/Button/Button.tsx

```diff
@@ -111,7 +112,11 @@ function ButtonComponent({
         />
       )}
 
-      <Focus parent={ButtonStyled} radius={radius} />
+      <Focus
+        parent={ButtonStyled}
+        radius={radius}
+        isKeyboardOnly={isKeyboardOnlyFocus}
```

Comment:
I don't know that we need a prop for this. I think we should just set this to true and remove the prop.

Reply:
Ah okay. That makes sense to me. 👍

Reply:
Sure, removing